File: Problem-1/SumOfEven.c
// WAP to find a sum of even number into 1D array.

#include <stdio.h>

int main() {
    int n, sum = 0;

    printf("Enter the size of array : ");
    scanf("%d", &n);

    int arr[n];

    printf("Enter elements into array :\n");
    for(int i = 0; i < n; i++) {
        printf("arr[%d] : ", i);
        scanf("%d", &arr[i]);
    }

    for(int i = 0; i < n; i++) {
        if(arr[i] % 2 == 0) {
            sum += arr[i];
        }
    }

    printf("The sum of even numbers : %d\n", sum);

    return 0;
}

File: Problem-1/SumOfEven.java
// WAP to find a sum of even number into 1D array.

import java.util.*;

public class SumOfEven{
    public static void main(String [] args){

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter the size of array : ");
        int n = sc.nextInt();

        int[] arr = new int[n];
        int sum = 0;

        System.out.println("Enter elements into array : ");
        for(int i = 0; i < n; i++){
            System.out.println("arr["+i+"] : ");
            arr[i] = sc.nextInt();
        }

        for(int i : arr){
            if(i % 2 == 0){
                sum += i;
            }
        }

        System.out.println("The sum of even numbers : "+ sum);
    }
}